internal/engine/analytics_nudge.go

```diff
@@ -0,0 +1,39 @@
+package engine
+
+import (
+	"os"
+
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/wmclient/pkg/analytics"
+)
+
+const newAnalyticsFlag = "TILT_NEW_ANALYTICS"
+
+// TEMPORARY: check env to see if new-analytics flag is set
+func newAnalyticsOn() bool {
+	return os.Getenv(newAnalyticsFlag) != ""
+}
+
+func MaybeSetNeedsNudge(mt *store.ManifestTarget, st *store.EngineState) {
```

Comment:
what is this manifest target supposed to be? as a caller, i'm not sure i know what I'd need to pass in here

Reply:
totally fair!